public/js/Horoscope.js

```diff
@@ -163,133 +145,52 @@ var Horoscope = function(args){
         //finds appropriate sign according to signCalendar.json data
         if (moment(date).format() !== "Invalid date"){
             if (moment(date).format("MMM") in this.calendar) {
-                let first_in_month = this.calendar[moment(date).format("MMM")]["first"];
-                let second_in_month = this.calendar[moment(date).format("MMM")]["second"];
-                let split = this.calendar[moment(date).format("MMM")]["divide"];
+                var first_in_month = this.calendar[moment(date).format("MMM")]["first"];
```

Comment:
lolzzzzz `let` pulls up an error if you're not in strict mode, so I elected to just avoid it where it isn't necessary for scope things. I only used it in the first place because I was excited that it existed, which is a bad reason to write any code :P